src/main/webapp/i18n/de/student-dashboard.json

```diff
@@ -197,7 +197,13 @@
                 "complaintDue": "Beschwerde bis:",
                 "complaintDueTooltip": "Du kannst eine Beschwerde bis {{date}} einreichen.",
                 "complaintPossible": "Beschwerde möglich:",
-                "complaintPossibleTooltip": "Nach Erhalt einer Bewertung wirst du {{days}} Tage zum Einreichen einer Beschwerde haben."
+                "complaintPossibleTooltip": "Nach Erhalt einer Bewertung wirst du {{days}} Tage zum Einreichen einer Beschwerde haben.",
+                "chatWelcome": "Willkommen",
+                "chatPopUpMessage": "Nachdem Sie die Bestimmungen akzeptiert haben, beginnt Ihr Chat mit IrisBot. Bitte lehnen Sie ab, wenn Sie nicht chatten möchten.",
```

Comment:
Imo this message should also inform a user about future transmission of their data to third-party servers.

Reply:
Done